### ISSUE TITLE
nyu_reduced evaluation - testset (line 75)

### ISSUE THREAD

**Issue by LuizGuzzo** [mic-rud/GuidedDecoding#8]
Hello, this work is very good! I am trying to test it but I am having some difficulties.
I have already managed to train the entire network and the accuracy during training is close to what was written in the paper, but when I try to test the trained model in the nyu_reduced.py code
on line 75 - "self.rgb = torch.from_numpy(np.load(BytesIO(data['eigen_test_rgb.npy']))).type(torch.float32) #Range [0,1]"
gives a key error, which made me think that this .npy might be from another dataset configuration. I ended up downloading the evaluation dataset that they made available in the README, but it does not contain the RGB and Depth in the same zip as Nyu_reduced, which left me a bit confused.
I would like to know what I would need to do to get these rgb and depth for the test. And also where this evaluation .zip came from, I am a bit new .
Is the evaluation dataset the test set of NYU? If so, does it come with the training dataset in the same .zip file? Or is it a separate set of images, like a train, validation, and test split?

**Comment by mic-rud** [comment]
The code was faulty and is now updated. The wrong dataloader was selected by default. 

The data is just the nyu Eigen test split. Each *.npz file contains image and depth values.
